app/src/main/java/com/example/coen268/CreateAccountActivity.java

```diff
@@ -39,6 +39,8 @@
     private FirestoreService firestoreService;
     private boolean mBound;
 
+    DatabaseReference m_databaseFcmRegToken;
```

Comment:
What is this reference for?

Reply:
Sorry, this should not be needed anymore. I'll fix it